examples/jsm/loaders/obj2/OBJLoader2Parser.js

```diff
@@ -367,7 +367,7 @@ OBJLoader2Parser.prototype = {
 
 				// LF
 				case 10:
-					this._processLine( buffer, bufferPointer, slashesCount, word, currentByte);
+					this._processLine( buffer, bufferPointer, slashesCount, word, currentByte );
```

Comment:
Welcome! Personal preference, but I think it is best to keep a PR focused, and save this for a separate "clean up" PR.

Reply:
I thought I would fix it since I ran the linter but can remove it.